moonshot/__main__.py

```diff
@@ -91,80 +91,117 @@ def moonshot_data_installation():
     folder_name = repo.split("/")[-1].replace(".git", "")
 
     # Check if the directory already exists
-    if not os.path.exists(folder_name):
-        logger.info(f"Cloning {repo}")
-        # Clone the repository
-        run_subprocess(["git", "clone", repo], check=True)
-
-        # Create .env to point to installed folder
-        ms_lib_env_file(folder_name)
-
-        # Change directory to the folder
-        os.chdir(folder_name)
-
-        logger.info(f"Installing requirements for {folder_name}")
-        # Install the requirements if they exist
-        if os.path.exists("requirements.txt"):
-            run_subprocess(["pip", "install", "-r", "requirements.txt"], check=True)
-            import nltk
-
-            nltk.download("punkt")
-            nltk.download("stopwords")
-            nltk.download("averaged_perceptron_tagger")
-            nltk.download("universal_tagset")
-
-        # Change back to the base directory
-        os.chdir("..")
-
-    else:
-        logger.warning(f"Directory {folder_name} already exists, skipping clone.")
+    if os.path.exists(folder_name):
+        logger.warning(f"Directory {folder_name} already exists.")
+        user_input = input(
+            f"Directory {folder_name} already exists. Do you want to remove it and reinstall? (Y/n): "
+        ).strip()
+        if user_input == "Y":
+            logger.info(f"Removing directory {folder_name}.")
+            # Remove the existing directory
+            run_subprocess(["rm", "-rf", folder_name], check=True)
```

Comment:
this is a bit dangerous with -rf. can we not do a force. but instead maybe have the user put a y or a n with the deletion instead.

Reply:
not sure if this can be merged with the check above